spec/lib/flipper/utilities/bulk_feature_checker_spec.rb

```diff
@@ -115,26 +115,26 @@
       end
     end
 
-    context 'when provided feature keys are strings' do
-      let(:enabled_features) { %w[enabled_feature5 enabled_feature6] }
-      let(:disabled_features) { %w[disabled_feature5 disabled_feature6] }
-      let(:missing_features) { %w[missing_feature5 missing_feature6] }
+    context 'when provided feature keys are symbols' do
+      let(:enabled_features) { %i[enabled_feature5 enabled_feature6] }
+      let(:disabled_features) { %i[disabled_feature5 disabled_feature6] }
+      let(:missing_features) { %i[missing_feature5 missing_feature6] }
       let(:features) { enabled_features + disabled_features + missing_features }
       let(:expected_result) do
         {
-          enabled: enabled_features.map(&:to_sym),
-          disabled: disabled_features.map(&:to_sym),
-          missing: missing_features.map(&:to_sym)
+          enabled: enabled_features.map(&:to_s),
+          disabled: disabled_features.map(&:to_s),
+          missing: missing_features.map(&:to_s)
         }
       end
 
       before do
         enabled_features.each do |feature|
-          Flipper.enable(feature.to_sym)
+          Flipper.enable(feature.to_s)
```

Comment:
The `to_s` is needed here because enabling the feature using a symbol causes `Flipper.features` to return only _this_ feature name as a symbol and the rest of the features as strings. This behavior of `Flipper.enable` only occurs in the testing environment, not in the production environment.